nettests/web_connectivity.go

```diff
@@ -3,30 +3,30 @@ package nettests
 import (
 	"github.com/apex/log"
 	"github.com/ooni/probe-cli/internal/database"
+	engine "github.com/ooni/probe-engine"
```

Comment:
Between v0.2.0 and v0.3.0 the ooni/probe-engine API changed. This adapts the code to the new API.